WalletWasabi/WabiSabi/Client/GreedyDecomposer.cs

```diff
@@ -0,0 +1,31 @@
+using System.Linq;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System;
+
+namespace WalletWasabi.WabiSabi.Client
+{
+	public class GreedyDecomposer
+	{
+		public GreedyDecomposer(IEnumerable<long> denominations)
```

Comment:
If we are sure that these are always sats, we could use Money to make it obvious.